Firestore/core/src/firebase/firestore/local/leveldb_transaction.cc

```diff
@@ -155,38 +130,78 @@ bool LevelDBTransaction::Iterator::Valid() {
   return is_valid_;
 }
 
+LevelDBTransaction::LevelDBTransaction(DB* db,
+                                       const ReadOptions& readOptions,
```

Comment:
snake_case.

Please review everything matching `\b[a-z]+[A-Z]` in the C++ files.

Reply:
Ok, I think I got them all.